docs/core/howto/tap.rst

```diff
@@ -310,6 +310,107 @@ For a full list of cred plugins supported, see :api:`twisted.plugins <twisted.pl
 
 
 
+Deploy your application using python packages
```

Comment:
titlecase section headings I think

Reply:
That is possible, however the unchanged section headings do not use titlecase. For consistency I did not use it, too.